requests_toolbelt/adapters/source.py

```diff
@@ -54,6 +54,15 @@ def __init__(self, source_address, **kwargs):
 
         super(SourceAddressAdapter, self).__init__(**kwargs)
 
+    def __getstate__(self):
```

Comment:
Requests defines [`__attrs__`](https://github.com/requests/requests/blob/8da118dfeb15989c47241d6ad6b7c429a34a51f3/requests/adapters.py#L108) which we could extend here, e.g.,

```py
__attrs__ = HTTPAdapter.__attrs__ + ['source_address']
```